Update README with some links to airgun/nailgun/ssh2-python (#8347)
SatelliteQE/robottelo: README.rst

automated, suited for use in a continuous integration environment, and `data
driven`_. There are three types of tests:

* UI tests, which rely on Selenium's `WebDriver`_ through `airgun`_.

> * CLI tests, which rely on `Paramiko`_ - and are being converted to using `ssh2-python`_.

Reviewer: ```suggestion
* CLI tests, which currently rely on `Paramiko`_ , but are being converted to use `ssh2-python`_.
```
Please avoid dashes and slashes in text
